### PR TITLE
11.0 web drop target add multi upload

### PR DESCRIPTION
**Pull request by etobella**
This PR Migrates all the changes from 10.0 to 11.0.
It is the same than #975 but also fixes an issue. On 11.0 if you had two different areas where you can drop files, it was attached to both, but you may be selecting just one. Now this change fixes it.
I decided to ignore #975 because no answers from the contributor were raised.

@jarroyomorales @jbeficent

### CODE REVIEW

File: web_drop_target/static/src/less/web_drop_target.less [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        pointer-events: none;
+        .o_drag_over_content{
+            position: relative;
+            top: 50%;
+            transform: translate(0%, -50%);
+            width: 100%;
+            text-align: center;
+        }
+    }
+}
```
Comment (jarroyomorales) [inline web_drop_target/static/src/less/web_drop_target.less]: Blank line

**Review by fuentes73 — APPROVED** [review]
Tested on runbot

**Review by jarroyomorales — APPROVED** [review]

**Comment by grindtildeath** [comment]
@etobella "I decided to ignore #975 because no answers from the contributor were raised"

Please ping the contributor before writing tales...

I have no issue in merging this one instead of #975 but can you please at least cherry-pick this commit as well ? https://github.com/OCA/web/pull/975/commits/3150f72a77bf2d4e325cddd4018f98e241471617

**Comment by etobella** [comment]
Well, there was a ping from @simahawk on 4th March and no answer, so I thought you were no longer working on it. Sorry for the misunderstanding, next time I will ping the contributor before.

Your commit has been added. Do you prefer to add my commit to your branch or we try to merge this one?

**Comment by grindtildeath** [comment]
@etobella As written I have no issue to merge this one as it's been a long time I haven't work on it, and this one is more complete :wink: 
So I will close the other one now.

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: technical and functional review.

File: web_drop_target/static/src/js/web_drop_target.js [hunk 2]
```diff
@@ -4,16 +4,17 @@
 
 odoo.define('web_drop_target', function(require) {
     var    FormController = require('web.FormController');
```
Comment (Tardo) [inline web_drop_target/static/src/js/web_drop_target.js]: Please, remove spaces

File: web_drop_target/static/src/js/web_drop_target.js [hunk 3]
```diff
@@ -4,16 +4,17 @@
...
+    var    core = require('web.core');
```
Comment (Tardo) [inline web_drop_target/static/src/js/web_drop_target.js]: Please, remove spaces

File: web_drop_target/static/src/js/web_drop_target.js [hunk 4]
```diff
@@ -103,6 +102,51 @@ odoo.define('web_drop_target', function(require) {
...
+        _file_reader_error_handler: function(e){
+            console.error(e);
+        },
+
+        _handle_file_drop_attach: function(
+            item, e, res_model, res_id, extra_data
+        ) {
+            var self = this;
+            var file = item;
+            if(!file || !(file instanceof Blob)) {
```
Comment (Tardo) [inline web_drop_target/static/src/js/web_drop_target.js]: Add space after 'if'

File: web_drop_target/static/src/js/web_drop_target.js [hunk 5]
```diff
@@ -103,6 +102,51 @@ odoo.define('web_drop_target', function(require) {
...
+            var reader = new FileReader();
+            reader.onloadend = self.proxy(
+                _.partial(self._create_attachment, file, reader, e, res_model, res_id, extra_data)
+            );
+            reader.onerror = self.proxy('_file_reader_error_handler');
+            reader.readAsArrayBuffer(file);
+        },
+
+        _add_overlay: function() {
+            if(!this._drop_overlay){
```
Comment (Tardo) [inline web_drop_target/static/src/js/web_drop_target.js]: Add space after "if"

**Review by Tardo — APPROVED** [review]
LGTM :+1:, tested on runbot. Only some non-blocking changes...

**Comment by pedrobaeza** [comment]
@etobella are you going to fix JS suggestions?

**Comment by etobella** [comment]
@pedrobaeza @Tardo Comments attended. Sorry for the delay

**Comment by etobella** [comment]
@pedrobaeza I think we can merge this one :wink:

**Comment by pedrobaeza** [comment]
Is this module still needed in v12, isn't it?

**Comment by etobella** [comment]
Probably, but no one started the migration so far

**Pull request closed without merging**